src/plmlists.asm

```diff
@@ -332,6 +344,14 @@ if !AREA == 1
     skip 42                 ; open the back door
     dw $0002 : skip 4
 endif
+
+; Phantoon's Boss Room - Room $CD13
+org $8FC2B3
+skip 4
+dw $8086                ; Set door as always solid grey so each version
+                        ; of the boss room is locked for true boss rando.
+dw $0000
```

Comment:
We can avoid the `dw $0000` because that's what is already in vanilla for those bytes.